plans/variables.tf

```diff
@@ -3,3 +3,4 @@ variable "client_id" {}
 variable "client_secret" {}
 variable "tenant_id" {}
 variable "prefix" {}
+variable "location" {}
```

Comment:
I have a similar commit which defines this, I think the variable should define a default, e.g.

```
variable "location" {
    type    = "string"
    default = "East US 2"
}
```